README.md

```diff
@@ -76,6 +76,8 @@ By default, the Jenkins slaves are run in the default Mesos container. To run th
 
 	2) "Use External Containerizer" : Select this option if Mesos slave(s) are configured with "--containerizers=external".
```

Comment:
I would just rephrase this instead

2) "Use Mesos Containerizer": Select this option if Mesos slave(s) are configured with "--containerizers=mesos"